src/engine/enginebuffer.cpp

```diff
@@ -493,7 +493,7 @@ void EngineBuffer::slotTrackLoading() {
 }
 
 TrackPointer EngineBuffer::loadFakeTrack(double filebpm) {
-    TrackPointer pTrack(new TrackInfoObject(), &QObject::deleteLater);
+    TrackPointer pTrack(TrackInfoObject::newTemporary());
```

Comment:
We can easily restore the no-argument version newDummy() for fake TIOs and 
require that newTemporary() is always invoked with at least the QFileInfo 
arg. They are handled in the same way so I decided to throw away newDummy() 
later.

On 08/31/2015 12:36 AM, Daniel Schürmann wrote:
>
> In src/engine/enginebuffer.cpp 
> <https://github.com/mixxxdj/mixxx/pull/695#discussion_r38276940>:
>
> > @@ -493,7 +493,7 @@ void EngineBuffer::slotTrackLoading() {
> >  }
> >
> >  TrackPointer EngineBuffer::loadFakeTrack(double filebpm) {
> > -    TrackPointer pTrack(new TrackInfoObject(), &QObject::deleteLater);
> > +    TrackPointer pTrack(TrackInfoObject::newTemporary());
>
> I think ther is a naming missmatch FakeTrack vs Temporary.
> Is this temorary only used for testing? Then "Dummy", which is was in one 
> of your earlyer commits will work well for me.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/695/files#r38276940>.
>